t/43-scheduling-and-worker-scalability.t

```diff
@@ -99,6 +99,7 @@ note("Spawning $worker_count workers");
 sub spawn_worker {
     my ($instance) = @_;
 
+    local $ENV{PERL5OPT} = '';                                             # uncoverable statement
```

Comment:
I guess we also want a source file comment for *that*. The alternative of course is always an explanatory function name

```suggestion
sub disable coverage_collection_in_spawned_processes {
    local $ENV{PERL5OPT} = '';                                             # uncoverable statement
}
…
sub spawn_worker {
…
    disable coverage_collection_in_spawned_processes;
```

Reply:
I like the separate function, makes it more obvious. Especially if we do the same in more places.